.buildkite/pipeline.yml

```
@@ -1,15 +1,13 @@
 agents:
   queue: central
```

```suggestion
  queue: new-central
```

I approved but we need this.

---

argh! Thank you